spec/integration/create_a_document_spec.rb

```diff
@@ -12,45 +13,66 @@
     and_the_preview_creation_succeeded
   end
 
+  scenario "User create a document and the chosen base path is already in use" do
```

Comment:
I wonder if we should pull this out into its own feature test (file). We currently have to do setups in this test (`when_i_click_on_create_a_document` / `and_i_choose_news `) that are unrelated to the thing that we're attempting to test (the base path).